### PR TITLE
enhancement: foundry tab

### LINKED ISSUE #5189: [Task]: Add tabs to minting / foundry activity
### Task description

We want to add a tabbed approach to the foundry minting activity, this will include a basic transaction information tab, a foundry specific tab and then a tab for the metadata (IRC30)

### Requirements

Add tabs to foundry activity

### Creation checklist

- [ ] I have assigned this task to the correct people
- [X] I have added the most appropriate labels
- [X] I have linked the correct milestone and/or project

### PR DESCRIPTION
**Pull request by MarkNerdi**
## Summary

Add foundry information tab to ActivityDetails

## Changelog

```
- Add foundry information tab to ActivityDetails
```

## Relevant Issues

Closes #5189 

### Platforms

> Please select any platforms where your changes have been tested.

- __Desktop__
  - [ ] MacOS
  - [ ] Linux
  - [ ] Windows
- __Mobile__
  - [ ] iOS
  - [ ] Android

### Instructions

> Please describe the specific instructions, configurations, and/or test cases necessary to __test and verify__ that your changes work as intended.

...

## Checklist

> Please tick all of the following boxes that are relevant to your changes.

- [ ] I have followed the contribution guidelines for this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added or modified tests that prove my changes work as intended
- [ ] I have verified that new and existing unit tests pass locally with my changes
- [ ] I have verified that my latest changes pass CI workflows for testing and linting
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

**Review by nicole-obrien — CHANGES_REQUESTED** [review]
Because you can mint a foundry without it following irc30. We should have three tabs:

- Transaction
- Foundry
  - Alias owner
  - Total supply
  - Circulating supply
  - Melted supply
- Token Metadata
  - If follows IRC30 we parse as key values etc
  - else we just show a metadata block: ideally formatted

**Comment by MarkNerdi** [comment]
Currently we don't yet support non-IRC30 tokens. But I'm gonna implement the 3rd tab with the assumption for now that every token is IRC30.

File: packages/shared/locales/en.json [hunk 1]
```diff
@@ -913,22 +913,29 @@
                 "nft": "NFT"
             }
         },
-        "mintNativeToken": {
```
Comment (nicole-obrien) [inline packages/shared/locales/en.json:916]: if you change this then every language needs to be re translated, unless you change the key in all language files

**Review by nicole-obrien — CHANGES_REQUESTED** [review]

**Review by nicole-obrien — APPROVED** [review]
LGTM!

**Pull request merged**